### DISCUSSION TITLE
Error building from sources: The specified path, file name, or both are too long.

### DISCUSSION THREAD

**Discussion by quanterion** [Dotnet-Boxed/Templates#30] [Q&A]
I downloaded your sources and while building Boilerplate.vsix project I get this error:
D:\Boilerplate\Source\packages\TemplateBuilder.1.1.4.4-beta\tools\ligershark.templates.targets(532,5): error MSB3021: Unable to copy file "D:\Boilerplate\Source\MVC6\Boilerplate.Web.Mvc6.Sample\node_modules\gulp\node_modules\liftoff\node_modules\findup-sync\node_modules\glob\node_modules\minimatch\node_modules\brace-expansion\package.json" to "obj\Debug\ls-Templates\ProjectTemplate\Boilerplate.Web.Mvc6.Sample.project\node_modules\gulp\node_modules\liftoff\node_modules\findup-sync\node_modules\glob\node_modules\minimatch\node_modules\brace-expansion\package.json". The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters.

**Comment by RehanSaeed** [comment]
Please see the [HOW TO BUILD](https://github.com/RehanSaeed/ASP.NET-MVC-Boilerplate/blob/master/HOW%20TO%20BUILD.md) document. It explains everything.
